utkdigitalinitiatives/TRACE
# address TRAC-922: rename (`git mv`) 095_ script to 900_ script

** JIRA Ticket**: [TRAC-922](https://jira.lib.utk.edu/browse/TRAC-922)

# What does this Pull Request do?
This PR attempts to address an occasional (or frequent) issue with Solr metadata display profiles; i.e. visiting `http://localhost:8000/admin/islandora/search/islandora_solr/settings` or `http://localhost:8000/admin/islandora/search/islandora_solr/metadata` yields empty solr fields under default display settings or an empty field configuration respectively.

These empty profiles, etc, are a result of the `095_load_solr_metadata_config_data` shell not executing correctly, or being overridden somewhere else as part of `vagrant up`, or some other problem.

# What's new?
`git mv`'d the filename from `095_...` to `900_...`

# How should this be tested?

1. Checkout this branch
2. `vagrant destroy --force`
3. `vagrant up`
4. After the VM has finished booting, visit `http://localhost:8000/admin/islandora/search/islandora_solr/settings` and `http://localhost:8000/admin/islandora/search/islandora_solr/metadata` and make sure that the display fields and/or field configuration settings are populated.

# Additional Notes:
This PR addresses a problem that seems to be intermittent. If you're uncertain about the veracity of this PR, you might try some sequences of: `vagrant up` (on the master branch - not this feature branch), visiting the Solr config pages above to review them, and then a `vagrant destroy --force`, repeat.

Questions/comments are welcome.

# Interested parties
@markpbaggett @cdeaneGit
